app/views/invite_requests/new.html.erb

```diff
@@ -1,27 +0,0 @@
-<!--Descriptive page name, messages and instructions-->
-<h2 class="heading">New invite_request</h2>
```

Comment:
Would you want to disable the route `new_invite_request` here, or leave that for [AO3-4972](https://otwarchive.atlassian.net/browse/AO3-4972)?

Reply:
Oh, I didn't even realize that issue existed! I'll just remove this change and let whoever takes that issue deal with it.

Reply:
I was kinda hoping you'd make it a double-issue pull request. :P

Reply:
If only I were slightly less sleep-deprived, I would be game! Maybe I'll make a separate PR tomorrow.